src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,82 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    public static final String OPERATION_VALUE_DELIMITER = ",";
+    public static final String SUPPLY_OPERATION_TITLE = "supply";
+    public static final String BUY_OPERATION_TITLE = "buy";
+    public static final String RESULT_OPERATION_TITLE = "result";
+    public static final int OPERATION_INDEX = 0;
+    public static final int VALUE_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeFile(composeReport(calculateOperations(readLinesFromFile(fromFileName))), toFileName);
+    }
```

Comment:
According to the checklist, large methods are difficult to read and understand, and methods should be short and responsible for one task. The method `getStatistic` is doing multiple things: reading from a file, calculating operations, composing a report, and writing to a file. Consider breaking down this method into smaller ones, each responsible for a single task.